### ISSUE TITLE
Text (editor) field at overpass-turbo.eu does not work correctly

### ISSUE THREAD

**Issue by sommerluk** [tyrasd/overpass-turbo#374]
Text (editor) field at overpass-turbo.eu does not work correctly

Steps to reproduce:

- Open Firefox
- Go to `overpass-turbo.eu`
- Copy and paste the following query:
```
[out:json][timeout:25];
node
  [name~".*[𠮷𠮹].*"]
  ({{bbox}});
out;
```
- Position the text cursor in line 3 between `]` and `.`.
- Push the left-arrow key on your keyboard to move the text cursor between `𠮹` and `]`.
- Push the backspace key on your keyboard.

Expected behaviour: the character `𠮹` is deleted.

Actual behaviour: a strange character appears at the place of the character `𠮹`.

I guess this has something to do with an UTF-16 decoding gone wrong somewhere?

Note: In JOSM, when downloading data using an overpass-turbo query, the text editing works fine.

**Comment by mmd-osm** [comment]
Seems like an upstream issues in the CodeMirror syntax highlighting library.
You can deactivate CodeMirror in _Settings -> Editor -> Deactivate CodeMirror_ in the meantime.

**Comment by tyrasd** [comment]
> CodeMirror

yes, but newer versions don't have this bug anymore. I tried upgrading, but it looks a bit complicated with our current setup (es6 modules + webpack). For example, I couldn't get the `multiplexingMode` addon to work in the newest version of CodeMirror included via webpack. :unamused:

**Comment by pyrog** [comment]
No issue under Safari, but I could reproduce it under Chrome.

With Firefox, the square appear broken when the source is pasted:
<img width="190" alt="Firefox screen copy" src="https://user-images.githubusercontent.com/2158081/70538543-9217ce00-1b62-11ea-9c7d-6daeb9a946e9.png">
